.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,128 @@
+name: CI/CD
```

Comment:
Any reason why you combined these?

Reply:
Because of `needs`, so they're in the same `jobs`